src/Microsoft.Health.SqlServer/DefaultSqlConnectionBuilder.cs

```diff
@@ -82,6 +82,23 @@ public async Task<SqlConnection> GetReadOnlySqlConnectionAsync(string initialCat
         return new SqlConnection(builder.ToString()) { RetryLogicProvider = _retryProvider };
     }
 
+    [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Caller must dispose result.")]
+    public async Task<SqlConnection> GetSqlConnectionAsync(bool isReadOnly, string applicationName)
```

Comment:
Will these connections never need the initialCatalog and maxPoolSize settings the other options allow for?

Reply:
I don't see any case when we would need master database or more than 100 connections for FHIR server.